src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String[] NAMES = {"supply", "buy", "result"};
+    private static final int INDEX_NAME = 0;
+    private static final int INDEX_AMOUNT_MONEY = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File files = new File(toFileName);
+        int[] results = new int[3];
+        if (files.exists()) {
+            return;
+        }
```

Comment:
What if the content of the file fromFileName will change? Or what if the file is empty? Let's better clear the file before writing new data.